object-manipulation-1/src/to-object.js

```diff
@@ -0,0 +1,10 @@
+/* exported toObject */
+function toObject(keyValuePair) {
+  const object = {};
+  const key = keyValuePair[0];
+  const value = keyValuePair[1];
```

Comment:
You'll learn more about this in the last week of junior side, but I'll give you a little preview. We can accomplish this in one line using something called "destructuring". Take a look at the snippet below!

`const [key, value] = keyValuePair`.

Obviously you do not need to start using this now, just wanted to show you what you can do! It will take out both values from the `keyValuePair` array and create 2 variables based off of what those 2 values are

Reply:
That's super cool! I can see how that would be super useful in a lot of situations.